server/schemas/OrderRequest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "OrderRequest",
+  "type": "object",
+  "properties": {
+    "address": {
+      "$ref": "RippleAddress"
+    },
+    "direction": {
```

Comment:
Finance terminology, hard to find a clear reference, but here is one: https://books.google.com/books?id=wmfpq6cwwcAC&pg=PA40&lpg=PA40&dq=trade+direction+buy+sell&source=bl&ots=CfanjD_9lf&sig=cgoiDOuUOU2fXC2QTIcTfHYxXpk&hl=en&sa=X&ei=BZEUVd2WD4quogSByYDADQ&ved=0CD8Q6AEwBA#v=onepage&q=trade%20direction%20buy%20sell&f=false